src/Cli.Tests/ValidateConfigTests.cs

```diff
@@ -319,24 +302,53 @@ public async Task TestValidateAKVOptionsWithoutEndpointFails()
     public async Task TestValidateAzureLogAnalyticsOptionsWithoutAuthFails()
     {
         // Arrange
+        ConfigureOptions options = new(
+            azureLogAnalyticsEnabled: CliBool.True,
+            azureLogAnalyticsDabIdentifier: "dab-identifier-test",
+            azureLogAnalyticsFlushIntervalSeconds: 1,
+            config: TEST_RUNTIME_CONFIG_FILE
+        );
+
+        // Act
+        await TestValidatePropertyOptionsFails(options);
+    }
+
+    /// <summary>
+    /// Tests that validation fails when File Sink options are configured with an empty 'path' property.
+    /// </summary>
+    [TestMethod]
+    public async Task TestValidateFileSinkOptionsWithoutPathFails()
+    {
+        // Arrange
+        ConfigureOptions options = new(
+            fileSinkEnabled: CliBool.True,
+            fileSinkRollingInterval: nameof(RollingIntervalMode.Day),
+            fileSinkRetainedFileCountLimit: 1,
+            fileSinkFileSizeLimitBytes: 1024,
+            config: TEST_RUNTIME_CONFIG_FILE
+        );
+
+        // Act
+        await TestValidatePropertyOptionsFails(options);
+    }
+
+    /// <summary>
+    /// Base test that ensures properties with missing options fail validation.
```

Comment:
This is not a test in itself since it is not marked with the TestMethod attribute, but a helper function. Please rename the function to simply ValidatePropertyOptionFails and fix the summary here.